airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -212,6 +212,17 @@ class Stream(HttpStream, ABC):
     primary_key = None
     filter_old_records: bool = True
 
+    @property
+    @abstractmethod
+    def scopes(self) -> Set[str]:
+        """Set of required scopes. Users need to grant at least one of the scopes for the stream to be avaialble to them"""
+
+    def scope_is_granted(self, granted_scopes: Set[str]) -> bool:
+        if not self.scopes:
+            return True
+        else:
+            return len(self.scopes.intersection(granted_scopes)) > 0
```

Comment:
shouldn't this be `len(intersection) == len(granted_scopes)`?

Reply:
my understanding is we need one of the scopes:
> media_bridge.read or tickets or crm.schemas.custom.read or e-commerce or timeline or contacts
https://developers.hubspot.com/docs/api/crm/pipelines

Reply:
👍🏼